web/backend/test/routes/root.test.ts

```diff
@@ -119,27 +118,49 @@ test('runtime is running', async (t) => {
   assert.strictEqual(serviceInvalidOpenapi.statusCode, 500, 'service OpenAPI endpoint')
   assert.strictEqual(typeof serviceInvalidOpenapi.json().code, 'string')
 
-  const logs = await server.inject({
-    url: `/runtimes/${runtimePid}/logs`
-  })
-  assert.strictEqual(logs.statusCode, 200)
-
-  const result = logs.json<Log[]>()
-  assert.ok(result.some(({ msg }) => msg.includes('Starting the service')))
-  assert.ok(result.some(({ msg }) => msg.includes('Started the service')))
-  assert.ok(result.some(({ msg }) => msg.includes('Server listening at')))
-  assert.ok(result.some(({ msg }) => msg.includes('Platformatic is now listening')))
-
-  const [{ level, time, pid, hostname }] = result
-  assert.ok(typeof level, 'number')
-  assert.ok(typeof time, 'number')
-  assert.ok(typeof pid, 'number')
-  assert.ok(typeof hostname, 'string')
```

Comment:
you need these assertions down in the following test

Reply:
as above, I don't see how we can mock `RuntimeApiClient` from `@platformatic/control`

Reply:
done on 37bd0bf